### ISSUE TITLE
Runtime error

### ISSUE THREAD

**Issue by Simbud-7** [Kavinraja-G/node-gizmo#25]
**Describe the bug**
I got a panic runtime error when running. Strace shows ~/.kube/config is missing, which is true. I use a $KUBECONFIG with multiple contexts in it. Even with a single config file in $KUBECONFIG, nodegizmo crashes

**Reproduce steps**
run nodegizmo with no ~/.kube/config file but a $KUBECONFIG variable

**Screenshots**
```
nodegizmo                                                                        
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x130bd5d]

goroutine 1 [running]:
k8s.io/client-go/kubernetes.NewForConfig(0x6?)
	/home/runner/go/pkg/mod/k8s.io/client-go@v0.28.0/kubernetes/clientset.go:461 +0x1d
github.com/Kavinraja-G/node-gizmo/utils.k8sAuth()
	/home/runner/work/node-gizmo/node-gizmo/utils/config.go:36 +0x18
github.com/Kavinraja-G/node-gizmo/utils.InitConfig()
	/home/runner/work/node-gizmo/node-gizmo/utils/config.go:47 +0x13
github.com/Kavinraja-G/node-gizmo/cmd.init.0()
	/home/runner/work/node-gizmo/node-gizmo/cmd/kubectl-ng.go:10 +0xf
```

**Version**
v0.1.6
https://github.com/Kavinraja-G/node-gizmo/releases/download/v0.1.6/nodegizmo_v0.1.6_linux_amd64.tar.gz

**Additional context**
```
uname -a
Linux IMP-IT0295LX 6.5.0-13-generic #13-Ubuntu SMP PREEMPT_DYNAMIC Fri Nov  3 12:16:05 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux

strace nodegizmo
mmap(NULL, 262144, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, -1, 0) = 0x7f6f72bf4000
mmap(0xc000400000, 4194304, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_ANONYMOUS, -1, 0) = 0xc000400000
futex(0xc00007c948, FUTEX_WAKE_PRIVATE, 1) = 1
--- SIGURG {si_signo=SIGURG, si_code=SI_TKILL, si_pid=44561, si_uid=954003561} ---
rt_sigreturn({mask=[]})                 = 1
newfstatat(AT_FDCWD, "/home/XXXX/.kube/config", 0xc000334e08, 0) = -1 ENOENT (No such file or directory)
--- SIGSEGV {si_signo=SIGSEGV, si_code=SEGV_MAPERR, si_addr=NULL} ---
rt_sigreturn({mask=[]})                 = 0
nanosleep({tv_sec=0, tv_nsec=1000000}, NULL) = 0
nanosleep({tv_sec=0, tv_nsec=1000000}, NULL) = 0
write(2, "panic: ", 7panic: )                  = 7
write(2, "runtime error: invalid memory ad"..., 64runtime error: invalid memory address or nil pointer dereference) = 64
write(2, "\n", 1
)                       = 1
write(2, "[signal ", 8[signal )                 = 8
write(2, "SIGSEGV: segmentation violation", 31SIGSEGV: segmentation violation) = 31
write(2, " code=", 6 code=)                   = 6
write(2, "0x1", 30x1)                      = 3
write(2, " addr=", 6 addr=)                   = 6
write(2, "0x0", 30x0)                      = 3
write(2, " pc=", 4 pc=)                     = 4
write(2, "0x130bd5d", 90x130bd5d)                = 9
write(2, "]\n", 2]
```

**Comment by Kavinraja-G** [comment]
@Simbud-7 Thanks for reporting this. I'll look into it.

**Comment by Simbud-7** [comment]
Thanks for fixing ! :)

**Comment by Kavinraja-G** [comment]
@Simbud-7 Fix is now released in `v0.1.7` Please run `kubectl krew upgrade nodegizmo` if you are using krew, thanks!
